evaluator/src/test/kotlin/RuleTest.kt

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.evaluator
+
+import com.here.ort.model.Severity
+
+import io.kotlintest.matchers.haveSize
+import io.kotlintest.should
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.WordSpec
+
+class RuleTest : WordSpec() {
+    private val ruleSet = RuleSet(ortResult)
+    private val errorMessage = "error message"
+
+    private fun createRule() = object : Rule(ruleSet, "test") {
+        override val description = "test"
+        override fun errorSource() = name
+    }
+
+    init {
+        "hint" should {
+            "add an issue with the correct severity" {
```

Comment:
cane be named: add an issue with the hint severity

same for below tests

Reply:
@fviernau : Would be it ok to merge this as is and fix the remaining issues later? I barely have time to improve this, but I would rather have the tests in the current state than no tests for the rules API at all.